packages/teleport/src/Databases/Databases.tsx

```diff
@@ -90,3 +120,15 @@ export function Databases(props: State) {
     </FeatureBox>
   );
 }
+
+const emptyStateInfo: EmptyStateInfo = {
+  title: 'ADD YOUR FIRST DATABASE',
+  description:
```

Comment:
Consolidate access to databases running behind NAT, prevent data exfiltration, meet compliance requirements, and have complete visibility into access and behavior. Follow [the documentation](https://goteleport.com/docs/database-access/guides/) to get started.